common-lib/src/main/scala/com/gu/mediaservice/model/leases/MediaLease.scala

```diff
@@ -55,17 +53,13 @@ case class MediaLease(
     case _ => this
   }
 
-  private def withValidEndDateField: MediaLease = if (access == AllowSyndicationLease) {
-    this.copy(endDate = None) // an allow-syndication cannot end
```

Comment:
Minor: I actually prefer having things on newlines just because I can reliably put a breakpoint on them. Not a big deal though

Reply:
Already moved to a new line :)